src/webui/webapplication.cpp

```diff
@@ -243,6 +202,43 @@ void WebApplication::sendWebUIFile()
     sendFile(localPath);
 }
 
+void WebApplication::translateDocument(QString &data)
+{
+    const QRegularExpression regex("QBT_TR\\((([^\\)]|\\)(?!QBT_TR))+)\\)QBT_TR\\[CONTEXT=([a-zA-Z_][a-zA-Z0-9_]*)\\]");
```

Comment:
True question: Why isn't the regex like this? (look at the last capture group)
```
"QBT_TR\\((([^\\)]|\\)(?!QBT_TR))+)\\)QBT_TR\\[CONTEXT=([a-zA-Z0-9_]*)\\]"
```

Reply:
To follow C++ (and many other languages) identifier format: "identifier cannot start with number". But if you want we can use softer rules here.

Reply:
I see.
Just leave it as it is.